docs/configuration/settings.md

```diff
@@ -352,16 +352,16 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
+| kyuubi.metrics.json.interval    | PT5S       | How often should report metrics to JSON file                                                                                                                                                                                                                                                                                                                                                                                                                                                         | duration | 1.2.0 |
+| kyuubi.metrics.json.location    | metrics    | Where the JSON metrics file located                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | string   | 1.2.0 |
+| kyuubi.metrics.prometheus.path  | /metrics   | URI context path of prometheus metrics HTTP server                                                                                                                                                                                                                                                                                                                                                                                                                                                   | string   | 1.2.0 |
+| kyuubi.metrics.prometheus.port  | 10019      | Prometheus metrics HTTP server port                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | int      | 1.2.0 |
+| kyuubi.metrics.reporters        | PROMETHEUS | A comma-separated list for all metrics reporters<ul> <li>CONSOLE - ConsoleReporter which outputs measurements to CONSOLE periodically.</li> <li>JMX - JmxReporter which listens for new metrics and exposes them as MBeans.</li>  <li>JSON - JsonReporter which outputs measurements to json file periodically.</li> <li>PROMETHEUS - PrometheusReporter which exposes metrics in Prometheus format.</li> <li>SLF4J - Slf4jReporter which outputs measurements to system log periodically.</li></ul> | set      | 1.8.0 |
```

Comment:
the version seems incorrect, have you regenerated it?

Reply:
Thanks. already corrected to right version.